src/reference/asciidoc/endpoint-summary.adoc

```diff
@@ -223,3 +223,6 @@ The `<int:outbound-channel-adapter>` element lets you send data to a `void` meth
 As discussed in <<./gateway.adoc#gateway,Messaging Gateways>>, the `<int:gateway>` element lets any Java program invoke a messaging flow.
 Each of these works without requiring any source-level dependencies on Spring Integration.
 The equivalent of an outbound gateway in this context is using a service activator (see <<./service-activator.adoc#service-activator,Service Activator>>) to invoke a method that returns an `Object` of some kind.
+
+Starting with version `5.2.2`, all the inbound gateway can be configured with an `errorOnTimeout` boolean flag to throw a `MessageTimeoutException` when downstream reply doesn't come back during reply timeout.
```

Comment:
`s/gateway/gateways/`

...when **the** downstream flow doesn't return a reply during the reply timeout.
The timer is not started until the thread returns the gateway so usually is only useful when the downstream flow is asynchronous.